u_public.go

```diff
@@ -92,8 +163,9 @@ func (chs13 *clientHandshakeStateTLS13) toPublic13() *PubClientHandshakeState {
 	} else {
 		tls13State := TLS13OnlyState{
 			KeySharesParams: chs13.keySharesParams,
-			EcdheKey:        chs13.ecdheKey,
-			KEMKey:          chs13.kemKey.ToPublic(),
+			EcdheKey:        chs13.keyShareKeys.ecdhe,
+			KEMKey:          chs13.keyShareKeys.publicKEMKey(),
+			KeyShareKeys:    chs13.keyShareKeys.ToPublic(),
```

Comment:
Do we need to provide EchdeKey and KEMKey if KeyShareKeys is provided? If not we should only provide one

Reply:
EchdeKey and KEMKey removed.